docs/manual-EN/1.overview/1.concepts/2.nGQL-overview.md

```diff
@@ -33,11 +33,11 @@
   - Each vertex has a unique 64-bit (signed integer) ID (**VID**)
   - Each vertex can associate with multiple **tags**
 - **Edge** : A Link between two vertices
-  - Each edge can be uniquely identified by a tuple **<src_vid, dst_vid, edge_type, rank>**
+  - Each edge can be uniquely identified by a tuple **<src_vid, dst_vid, edge_type, rank
```

Comment:
why??

Reply:
a typo here, thanks, I want to delete all the htmls and fix the lints in this file, but I'm not familiar with BNF so I reserved changes.